app/javascript/shared/components/ChatForm.vue

```diff
@@ -90,6 +91,20 @@ export default {
       }
       this.$emit('submit', this.formValues);
     },
+
+    fieldValidation(name, regex = '') {
+      const formValue = this.formValues[name];
+      if (!regex) {
+        return true;
+      }
+      const [, pattern, flags] = regex.match(/\/(.*)\/([a-z]*)/);
+      const reg = new RegExp(pattern, flags);
```

Comment:
I didn't understand this part. Why do we need this? Can directly use new RegExp(regex)

Reply:
Changed this method to the input pattern attribute method.